Source/WebKit/UIProcess/mac/CorrectionPanel.mm

```diff
@@ -28,27 +28,12 @@
 
 #if USE(AUTOCORRECTION_PANEL)
 
+#import <WebCore/CorrectionIndicator.h>
```

Comment:
imports with <> go after imports with "", each in lexical order

Reply:
done.